rest-tests/pom.xml

```diff
@@ -182,6 +120,84 @@
           -Xrunjdwp:transport=dt_socket,address=8797,server=y,suspend=y</maven.surefire.debug>
       </properties>
     </profile>
+
+    <profile>
```

Comment:
This new profile is not mandatory: you can simply start individual tests (or the whole suite) from your IDE